app/experimenter/experiments/models.py

```diff
@@ -247,6 +247,8 @@ class Experiment(ExperimentConstants, models.Model):
         default=None, blank=True, null=True
     )
 
+    paused = models.BooleanField(default=False)
```

Comment:
I think in general it's a good practice to label boolean fields like this as `is_whatever`

Reply:
will do